dev-docs/source/GettingStartedWithPyCharm.rst

```diff
@@ -180,11 +181,32 @@ The ``pydevd`` package does not have to be installed on Python 2. As of 12/11/20
 Setting up PyCharm on Linux
 ###########################
 
-On Linux the instructions are identical to Windows except that :
+1. Use the native python interpreter (``/usr/bin/python2.7/python.exe``) rather than from ``<Mantid Source Directory>/external/src/ThirdParty/lib/python2.7/python.exe``
+2. In the ``Project Structure`` sub menu you should see your root directory with the source/build directories both visible (if not, add them). The folder structure should be present in the centre of the window allowing you to mark folders orange (excluded) or blue (source). Source directories will be searched for python code.
+
+   Within the source directory add the following to your sources:
+
+   .. code-block:: sh
+
+       <Mantid Source Directory>/Framework/PythonInterface
+       <Mantid Source Directory>/scripts
+       <Mantid Source Directory>/qt/applications/workbench
+       <Mantid Source Directory>/qt/widgets
+       <Mantid Source Directory>/qt/python
+
+
+   The first folder can be replaced with the folder that contains your code, if you aren't writing code in ``scripts/``.
```

Comment:
Should this be referring to the second folder?

Reply:
Good catch, I'll move the `scripts/` line to the top again instead though